pkg/apiclient/workflowarchive/workflow-archive.proto

```diff
@@ -41,4 +48,10 @@ service ArchivedWorkflowService {
     rpc ListArchivedWorkflowLabelValues (ListArchivedWorkflowLabelValuesRequest) returns (github.com.argoproj.argo_workflows.v3.pkg.apis.workflow.v1alpha1.LabelValues) {
         option (google.api.http).get = "/api/v1/archived-workflows-label-values";
     }
+    rpc RetryArchivedWorkflow (RetryArchivedWorkflowRequest) returns (github.com.argoproj.argo_workflows.v3.pkg.apis.workflow.v1alpha1.Workflow) {
+        option (google.api.http) = {
+			put: "/api/v1/archived-workflows/{namespace}/{uid}/retry"
```

Comment:
Should this request be `/api/v1/archived-workflows/{uid}/retry` instead since no other workflow archive request uses the namespace?

@terrytangyuan @sarabala1979